src/org/jitsi/impl/neomedia/rtcp/RTCPTCCPacket.java

```diff
@@ -539,21 +518,22 @@ public RTCPTCCPacket(long senderSSRC, long sourceSSRC,
             else if (seqDelta % 7 == 3)
```

Comment:
stuff like this seems very confusing, but i assume it's just how the spec defines things?